apps/rds_tx.grc

```diff
@@ -1445,6 +1492,10 @@
       <value>14</value>
     </param>
     <param>
+      <key>radiotext</key>
+      <value>GNU Radio &lt;3</value>
+    </param>
+    <param>
       <key>ta</key>
```

Comment:
I don't see where this is used. I tried with the loopback flow graph and, at least there, it doesn't seem to work